### PR TITLE
Restaurant JS - Webpack

### PR DESCRIPTION
**Pull request by edxco**
In this project, I created a Fictional Restaurant website with sections to navigate within.
I used modularized Javascript and WebPack as the JS bundler. 
Sections are loading completely with javascript.

I have used TailWindCss as a CSS Framework that can be found in the ./dist folder.
Index.js is the entry point for WebPack.

### CODE REVIEW

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work. 💻 👀 ✨ 
- Impressive use of animation and maps.


## However, there are some changes required 👀 🧰 🔨 🔧 
- Please fix this issue with the broken images on your menu page. It shows up when I visit your Live Demo version: 
image
- Also please add a `dev` option to your `package.json`:
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️ - Your project looks great.

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+// Functions for menus
+const goToIndex = () => {
+  mainDiv.textContent = '';
+  mainDiv.appendChild(home());
+};
+
+// const goToAbout = () => {
+//   mainDiv.textContent = '';
+//   mainDiv.appendChild(loadAboutUs());
+// };
```
Comment (krys2fa) [inline src/index.js]: - [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Excellent work on this project @edxco  👏 
Your PR title and description are well written. 💪 
You have a clean code base and the project looks great. 👍 

### Status: Changes Required ♻️ 

**NB:** See below for a change to implement to get your project approved.

Please request another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @edxco :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**